ibm/resource_ibm_pi_instance.go

```diff
@@ -125,6 +126,44 @@ func resourceIBMPIInstance() *schema.Resource {
 				Computed:    true,
 				Description: "Storage type for server deployment",
 			},
+			PIInstanceStoragePool: {
```

Comment:
All these are marked computed also...if we don't pass will API provides some value?
Set all these attributes in Read method

Reply:
Good catch. I will set it in read method.